learnyounode/01-baby-steps.js

```diff
@@ -0,0 +1,12 @@
+sumNum = function(array) {
```

Comment:
This needs to be prefaced by either `const` or (less commonly) `var`. If you don't, the function gets added to the global namespace.